src/OSPSuite.SimModel/include/SimModel/SimulationOptions.h

```diff
@@ -22,6 +22,8 @@ namespace SimModelNative
 		bool _keepXMLNodeAsString; //original xml is required only for saving the simulation to XML
 		bool _useFloatComparisonInUserOutputTimePoints; //if set to true, float comparison will be used
 		                                                //for user output time points.Otherwise: double
+		bool _identifyUsedParameters; //if set to false: ALL parameters will be marked as used in ODE variables/observes
```

Comment:
is there a reason why comments are not above the variable? Kind of unusual

Reply:
No arguable reason :)